logbook-core/src/main/java/org/zalando/logbook/QueryFilters.java

```diff
@@ -33,4 +32,10 @@ public static QueryFilter replaceQuery(final String name, final String replaceme
         return query -> pattern.matcher(query).replaceAll(replacementPattern);
     }
 
+    @API(status = MAINTAINED)
+    public static QueryFilter removeQuery(final String name) {
```

Comment:
Can you describe your use case a bit more?

Reply:
there is some cases, when request contains a lot of query parameters, but you not really interested in them. 
in same time it could be few parameters you would like to keep. So it may make some sense to omit some parameters from logging. currently we using `replaceQuery` and it works quite good so far, until query names are small. unfortunately it not scaling with long query parameter names.

Reply:
I see. That makes sense.